snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: classic
+summary: A classic shell environment for Ubuntu Core all-snap installations
+description: This snap provides a full classic shell environment including dpkg and apt support.
+version: 16.04-3
```

Comment:
i just copied what was existing in meta/snap.yaml, there is definitely no need for -3 there